foundation/foundation-maven-plugins/artifacts-maven-plugin/pom.xml

```diff
@@ -1,40 +1,41 @@
 <?xml version="1.0"?>
```

Comment:
I (Informational): fixes build resolution where cucumber is pulling a `-SNAPSHOT` version of the `io.cucumber:messages:19.1.4` and breaking the build (may work if you are pointing to a Maven mirror that pulls in SNAPSHOTs subtly - but will not work on strict public repos)